index.html

```diff
@@ -138,16 +138,38 @@ <h2 class="pa-h2-v2-hero-subhead pa-white">Let us do the technological and profe
     <!-- /.pa-hero -->
 
 
-    <!-- CTA Section
-	================================================== -->
-    <section class="pa-cta-v1" data-parallax="scroll" data-speed="0.2" data-image-src="assets/images/image-section.jpg">
-
-        <div class="container">
-            <h1 class="pa-h1-v1 pa-bright fadeInLeftDelay03Duration10"><strong>Coolest HTML Theme Yet?</strong></h1>
-            <h4 class="pa-h4-v1 pa-bright fadeInLeftDelay03Duration10">Open Particle today, why not!</h4>
-            <div class="mt-4"></div>
-            <a class="pa-button-primary pa-hvr-sweep-to-right fadeInRightDelay11Duration10" href="https://themeforest.net/item/particle-modern-tech-startup-html-template/20078383?ref=Skilltech" target="_blank">Get it Now</a>
+   <section  class="pa-hero pa-centered-section pa-full-height pa-image-back pushable-content" style="background-image: url(assets/images/web5.jpg);">-->
+	
+      <div class="container">
+    <div class="text-center">
+    <div class="row">
+        <div class="col-lg-12">
+            <h3 style="font-size:60px"> About Us </h3>
+        </div>
+    </div>
+    </div>
+
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-6">
+                <div class="container">
+                   <h1> We make our <br>people and products<br>sensational.</h1>
+                            <br>
+                            <p>To revolutionize the way people do software and delivering happiness to our stakeholders and society.</p>
+                            <br>
+                        </div>
+                        <a class="pa-button-secondary pa-hvr-sweep-to-right fadeInRightDelay11Duration10" href="" target="_blank">Schedule Your Free Consultation</a>    
+                    </div>
+            <div class="col-sm-6">
+               <br><br><br>
+                <p class="content">We are public benefit organization established when the world appears to be on cross roads in light of COVID-19 pandemic crisis.
```

Comment:
try to arrange the Header text and the Paragraph in equal line.
![image](https://user-images.githubusercontent.com/67912894/125897333-276795d5-41c8-42c3-94fb-41b719830550.png)

Reply:
done